### PR TITLE
Internal: Editor V2 - Allow loading a document without reloading the page [ED-10379]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/js/editor/components/documents/commands/open.js [hunk 1]
```diff
@@ -17,12 +17,20 @@ export class Open extends $e.modules.CommandBase {
 			elementor.$previewContents.find( `.elementor-${ id }` ).addClass( 'loading' );
 		}
 
+		if ( setAsInitial ) {
+			// Set the new initial document id to be sent with all requests.
```
Comment (Nevoss) [inline assets/dev/js/editor/components/documents/commands/open.js:21]: Why? Please explain that the next request uses the initial document id to return the preview URL that's why it has to be here and not when the document is loaded

File: assets/dev/js/editor/editor-base.js [hunk 2]
```diff
@@ -977,7 +977,7 @@ export default class EditorBase extends Marionette.Application {
 		// TODO: Should be command?
 		jQuery( '#elementor-preview-loading' ).show();
 
-		this.$preview[ 0 ].contentWindow.location.reload( true );
+		this.$preview[ 0 ].src = this.config.initial_document.urls.preview;
```
Comment (Nevoss) [inline assets/dev/js/editor/editor-base.js:980]: Please check the case when you are editing loop item that the initial document was some page and then switch the canvas view

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/components/documents/commands/open.js (hunk 1 above)
Comment (StyleShit) [reply assets/dev/js/editor/components/documents/commands/open.js:21]: Good point, done

File: assets/dev/js/editor/editor-base.js (hunk 2 above)
Comment (StyleShit) [reply assets/dev/js/editor/editor-base.js:980]: NO

**Review by Nevoss — APPROVED** [review]

**Pull request merged**